src/neo/Network/P2P/Payloads/PingPayload.cs

```diff
@@ -7,26 +7,29 @@ namespace Neo.Network.P2P.Payloads
     public class PingPayload : ISerializable
     {
         public uint LastBlockIndex;
+        public long LastStateIndex;
```

Comment:
I believe this is not needed, the `Index` of the State will follow the block, or at least, will be 1 `block` delayed. We can process a process without a delay of more than 1 State.